enhancements/sig-policy/89-operator-policy-kind/README.md

```diff
@@ -0,0 +1,310 @@
+   including the operator pod's health, with clear violation messages if something is not right.
+4. What happens when the policy is deleted should be configurable: the operator might be deleted, or
+   the operator and its CRDs might be deleted, or nothing might be deleted.
+
+### Non-Goals
+
+1. The installation of OLM on managed clusters will not be automated by this enhancement.
+2. Management of "Operands" will not be included in this new type - they should be defined in
+   separate `ConfigurationPolicies`, possibly with a
+   [dependency](../74-order-policy-execution/README.md) on the `OperatorPolicy`.
+3. Management of `CatalogSources` will not be included in the new type - they should be defined in
+   separate `ConfigurationPolicies`.
+
+## Proposal
+
+### OperatorPolicy Syntax
+
+```yaml
+apiVersion: policy.open-cluster-management.io/v1beta1
+kind: OperatorPolicy
+metadata:
+  name: my-operator
+spec:
+  remediationAction: enforce # or inform
+  severity: medium
+  operatorGroup: # optional
+    name: my-operator-group
+    namespace: own-namespace
+    target:
+      namespaces:
+        - foo
+        - bar
+    #   selector: # One of selector or namespaces is allowed.
+    #     matchLabels: # or matchExpressions
+    #       mydomain.io/prod: "true"
+    serviceAccountName: my-og-account # optional
+  subscription:
+    channel: stable
+    name: my-operator
+    namespace: own-namespace
+    source: my-catalog
+    sourceNamespace: my-catalog-namespace
+    startingCSV: my-operator.v0.1.0 # optional
+    endingCSV: Any # or None or e.g. 'my-operator.v0.2.0`
+  requireLatestVersion: false
+  pruneOnDeletion:
+    - subscriptions
+    - clusterserviceversions
+    - installplans
+    - customresourcedefinitions
+    - apiservicedefinitions
+```
+
+The optional `spec.operatorGroup` section defines an
+[OperatorGroup](https://github.com/operator-framework/api/blob/master/crds/operators.coreos.com_operatorgroups.yaml).
+If not specified, and if the operator supports this InstallMode, then an
+[AllNamespaces](https://olm.operatorframework.io/docs/advanced-tasks/operator-scoping-with-operatorgroups/#targetnamespaces-and-their-relationship-to-installmodes)
+OperatorGroup will be created.
+
+The `spec.subscription` section includes fields from the spec of an [OLM
+Subscription](https://github.com/operator-framework/api/blob/master/crds/operators.coreos.com_subscriptions.yaml),
+excluding `config` and `installPlanApproval`. The `endingCSV` field allows configuration of which
+InstallPlans will be approved by the controller when the policy is set to `enforce`.
+
+The `spec.requireLatestVersion` field configures whether the "latest" version of the operator
+defined in the catalog on this cluster "should" be installed. Note that if the "latest" version is
+greater than the allowed `endingCSV`, then that version will not be automatically approved even if
```

Comment:
> The spec.requireLatestVersion field configures whether the "latest" version of the operator defined in the catalog on this cluster "should" be installed.

Wouldn't it be the default behavior?

Reply:
I don't know what you mean.

One case I was trying to account for was where the policy was created with the (current) "newest" version available and tested, but not wanting to immediately update whenever something new comes out. Instead, the user could use this so that they are notified (via a NonCompliant policy) when an update is available, so they can test it out.